implementations/rust/ockam/ockam_command/src/identity/import.rs

```diff
@@ -0,0 +1,57 @@
+use crate::util::async_cmd;
+use crate::{fmt_ok, CommandGlobalOpts};
+use clap::Args;
+use colorful::Colorful;
+use miette::IntoDiagnostic;
+use ockam_api::{color_primary, random_name};
+
+/// Change the default identity
+#[derive(Clone, Debug, Args)]
+pub struct ImportCommand {
+    #[arg(hide_default_value = true, default_value_t = random_name())]
+    pub name: String,
+
+    /// Name of the vault to use
+    #[arg(long, value_name = "VAULT_NAME")]
+    vault: Option<String>,
+
+    /// Identity to import in hex format
+    #[arg(long, value_name = "IDENTITY")]
+    identity: String,
+}
+
+impl ImportCommand {
+    pub fn run(self, opts: CommandGlobalOpts) -> miette::Result<()> {
+        async_cmd(&self.name(), opts.clone(), |_ctx| async move {
+            self.async_run(opts).await
+        })
+    }
+
+    pub fn name(&self) -> String {
+        "identity import".into()
+    }
+
+    async fn async_run(&self, opts: CommandGlobalOpts) -> miette::Result<()> {
+        let named_vault = opts.state.get_named_vault_or_default(&self.vault).await?;
+        let vault = named_vault.vault().await?;
+        let identifier = opts
+            .state
+            .make_identities(vault)
+            .await?
+            .identities_verification()
```

Comment:
It can be instantiated with `SoftwareVaultForVerifyingSignatures`, unless there is purpose in initializing the full Vault instance here

Reply:
No, there isn't. I thought we had to use the AWS vault to verify the identity.

Reply:
No, AWS only implements `VaultForSigning` trait, it's not a dependency for that operation 🙂

Reply:
Got it, I've added that change.